src/main/java/rx/internal/operators/SingleOperatorOnErrorResumeNext.java

```diff
@@ -0,0 +1,68 @@
+package rx.internal.operators;
+
+import rx.Single;
+import rx.SingleSubscriber;
+import rx.exceptions.Exceptions;
+import rx.functions.Func1;
+import rx.plugins.RxJavaPlugins;
+
+public class SingleOperatorOnErrorResumeNext<T> implements Single.OnSubscribe<T> {
```

Comment:
Just renamed `SingleOperatorOnErrorResumeNextViaSingle` to something more general and it's now works with function instead of `Single` directly.